packages/tools/src/tools/segmentation/strategies/fillSphere.ts

```diff
@@ -100,9 +119,23 @@ export function fillInsideSphere(
   enabledElement: Types.IEnabledElement,
   operationData: OperationData
 ): void {
-  fillSphere(enabledElement, operationData, true);
+  SPHERE_STRATEGY.fill(enabledElement, operationData);
 }
 
+SPHERE_STRATEGY.assignMethods(fillInsideSphere);
```

Comment:
I kind of understood your api for new BrushStrategy, but this bit `SPHERE_STRATEGY.assignMethods(fillInsideSphere);` is quite unusually, can you work around this so that the strategy after creation doesn't need this assignMethods? It is a confusing api

```js
const SPHERE_STRATEGY = new BrushStrategy(
  'Sphere',
  initializeRegionFill,
  initializeSetValue,
  initializeSphere,
  initializeTracking
);

export function fillInsideSphere(
  enabledElement: Types.IEnabledElement,
  operationData: OperationData
): void {
  SPHERE_STRATEGY.fill(enabledElement, operationData);
}

SPHERE_STRATEGY.assignMethods(fillInsideSphere);
```

I thought initializeSetValue is acutally the fill